src/posts/spotlights/jordan.md

```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
 tags:
   - middle east
   - culture
+packing:
+  - Sunglasses, sunscreen and sun/wind protecting hat
+  - Warm clothes for the desert night
+  - Comfortable shoes
+  - A light backpack for carrying food and water for the day
 ---
 
-# Petra and Wadi Rum
+## Understand
 
-## Wow
+Jordan is a small country in the Middle East, often considered one of the most stable from the political and economic perspective. The country attracts more than 1 million visitors every year, most of whom are coming specially for the rock city of Petra, one of the [seven new wonders of the world](https://en.wikipedia.org/wiki/New7Wonders_of_the_World).
```

Comment:
"the most stable from the economic and political perspective"
"most of whom are coming especially"